src/deviation.jl

```diff
@@ -1,7 +1,11 @@
 # Computing deviation in a variety of ways
 
 ## count the number of equal/non-equal pairs
+"""
+    counteq(a::AbstractArray, b::AbstractArray)
 
+Count the number of equal pairs of elements in a and b, i.e countnz(a .== b).
```

Comment:
"i.e."

Also could be more explicit, like "i.e. equivalent to `countnz(...)` but faster".